CMP1903_A1_2324/Game.cs

```diff
@@ -6,16 +6,33 @@
 
 namespace CMP1903_A1_2324
 {
+    /// <summary>
+    /// This creates the dice roll.
+    /// </summary>
     internal class Game
     {
-        /*
-         * The Game class should create three die objects, roll them, sum and report the total of the three dice rolls.
-         *
-         * EXTRA: For extra requirements (these aren't required though), the dice rolls could be managed so that the
-         * rolls could be continous, and the totals and other statistics could be summarised for example.
-         */
+        private static Die newDice = new Die();
```

Comment:
Consider changing newDice to newDie (Die is singular, dice is plural)

Reply:
Done 👍🏻